UnityProject/Assets/Scripts/Chat/DiscordWebhook.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Specialized;
+using System.Net;
+using System.Text.RegularExpressions;
+
+
+namespace DiscordWebhook
+{
+	public static class DiscordWebhookMessage
+	{
+		public static byte[] Post(string url, NameValueCollection pairs)
+		{
+			using (WebClient webClient = new WebClient())
+				return webClient.UploadValues(url, pairs);
+		}
+
+		public static void SendWebHookMessage(string url, string msg, string username, string mentionID = null)
```

Comment:
I'd suggest refactoring this slightly.

Perhaps have an enum (or similar) of channels (OOC, Admin, etc.) to pass in rather than a url, and then have this method work out the appropriate url. You could then also pull the three repeated string.IsNullOrEmpty calls into here - that'd also abstract away the implementation detail of the URL from what the callers of the method really care about which is the channel, and allow you to make one message go to multiple channels in the future.

Reply:
Think ive done what youve suggested.